manual/physics_vectors/index.md

```diff
@@ -7,762 +7,50 @@ toc: true
 toc_sticky: true
 ---
 
-ROOT provides several packes for physics vectors:
+Manual: physics vectors
+=======================
```

Comment:
I think the heading is automatically added by the infrastructure, I only added it for the CodiMD